releasenotes/notes/28742.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: release-notes/v2
+kind: feature
+
+area: traffic-management
+
+issue:
+    - https://github.com/istio/istio/issues/28742
+
+releaseNotes:
+- |
+  **Added** Configuring mesh cluster as JWT issuer for mTLS and other benefits. This is disabled by default 
```

Comment:
same here, the description needs a bit update, see the other comment about the flag.

Reply:
Updated